treadmill/admin.py

```diff
@@ -391,8 +409,9 @@ def connect(self):
                 server = ldap3.Server(uri)
                 self.ldap = ldap3.Connection(
                     server,
-                    authentication=ldap3.SASL,
-                    sasl_mechanism='GSSAPI',
+                    authentication=ldap3.SIMPLE,
```

Comment:
Make the authentication options read from that `_TREADMILL_CONFIG_PATH`. i.e. either:
```
---
authentication: sasl
sasl_mechnism: gssapi
```

or

```
---
authentication: simple
user: foo
password: secret
```